### PR TITLE
[fastlane] Show custom action message only on verbose mode

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

File: fastlane/lib/fastlane/actions/actions_helper.rb [hunk 1]
```diff
@@ -87,7 +87,7 @@ def self.load_external_actions(path)
           class_ref = Fastlane::Actions.const_get(class_name)
 
           if class_ref.respond_to?(:run)
-            UI.success "Successfully loaded custom action '#{file}'."
+            UI.success "Successfully loaded custom action '#{file}'." if $verbose
```
Comment (mfurtak) [inline fastlane/lib/fastlane/actions/actions_helper.rb]: Does it need to be green? Otherwise, just use `UI.verbose`?

Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/actions_helper.rb]: It doesn't I guess. Should I change it? It's kind of a success message

Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/actions_helper.rb]: Since it says that the external action was successfully loaded and is a valid action

Comment (mfurtak) [inline fastlane/lib/fastlane/actions/actions_helper.rb]: OK, just thought I'd ask :smile:

**Comment by mfurtak** [comment]
:ship: it

**Pull request merged**